packages/web/src/lib/web-mode.ts

```diff
@@ -38,7 +40,7 @@ export class WebMode {
 				pendingInfo[CONFIRMED_ROUND] !== null &&
 				(pendingInfo[CONFIRMED_ROUND] as number) > 0
 			) {
-				return pendingInfo;
+				return pendingInfo as unknown as algosdk.modelsv2.PendingTransactionResponse;
```

Comment:
Can we only use as `algosdk.modelsv2.PendingTransactionResponse` at here?

Reply:
No we can't, there are some properties missing